openeo/rest/datacube.py

```diff
@@ -2724,9 +2724,8 @@ def sar_backscatter(
         .. versionadded:: 0.4.9
         .. versionchanged:: 0.4.10 replace `orthorectify` and `rtc` arguments with `coefficient`.
         """
-        coefficient_options = [
-            "beta0", "sigma0-ellipsoid", "sigma0-terrain", "gamma0-ellipsoid", "gamma0-terrain", None
-        ]
+        schema = self.connection.get_schema("sar_backscatter", "coefficient")
+        coefficient_options = search_list_for_dict_key(schema, "enum")
```

Comment:
There should be fallbacks for several situations:
- there is no connection (`self.connection` is None)
- the backend does not provide an enum with possible  coefficient values